guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/forms/promote/mock/GetFormParamsEventMock.java

```diff
@@ -0,0 +1,29 @@
+package org.guvnor.asset.management.client.editors.forms.promote.mock;
+
+import org.uberfire.ext.widgets.common.client.forms.GetFormParamsEvent;
+
+import javax.enterprise.event.Event;
+import java.lang.annotation.Annotation;
+
+public class GetFormParamsEventMock implements Event<GetFormParamsEvent> {
```

Comment:
We do have a general purpose CDI Event mock in uberfire-testing-utils: https://github.com/uberfire/uberfire/blob/master/uberfire-testing-utils/src/main/java/org/uberfire/mocks/EventSourceMock.java

It probably makes sense to use that instead of creating custom event mocks everywhere we need them. If you need to assert on the fired event you can use an argument captor to get to it.